src/js/utils/option-utils.jsx

```diff
@@ -171,3 +171,20 @@ export const debounceLocationGroupsFetch = (waitTime, minSearchLength) =>
       callback([]);
     }
   }, waitTime);
+
+export const organizationsFetch = (roleTypes = ['ROLE_SUPPLIER'], active = false) =>
+  apiClient.get(`/openboxes/api/organizations?${roleTypes ? roleTypes.map(roleType => `&roleType=${roleType}`).join('') : ''}&active=${active}`)
+    .then((res) => {
+      if (res.data.data) {
+        return res.data.data.map(obj => (
+          {
+            id: obj.id,
+            value: obj.id,
+            name: obj.name,
+            label: `${obj.name}`,
+          }
+        ));
+      }
+      return null;
+    });
+
```

Comment:
I saw we only had `debounceFetch` of organizations, so I added a new one, which is not debounced. I needed it for `Purchasing organization` fetch.